docker-compose.yml

```diff
@@ -13,19 +13,37 @@ services:
       - ./.compose/neo4j/logs:/logs
       - ./.compose/neo4j/plugins:/plugins
     environment:
-      # Raise memory limits
+      # Raise memory limits:
       - NEO4J_dbms_memory_pagecache_size=1G
       - NEO4J_dbms.memory.heap.initial_size=1G
       - NEO4J_dbms_memory_heap_max__size=1G
+      # Auth:
+      - NEO4J_AUTH=none
+      # Add APOC and GDS:
+      - apoc.export.file.enabled=true
+      - apoc.import.file.enabled=true
+      - apoc.import.file.use_neo4j_config=true
+      - NEO4JLABS_PLUGINS=["graph-data-science", "apoc"]
+      - NEO4J_dbms_security_procedures_allowlist=gds.*, apoc.*
+      - NEO4J_dbms_security_procedures_unrestricted=gds.*, apoc.*
+      # Networking:
+      - dbms.connector.bolt.listen_address=0.0.0.0:7687
+
     healthcheck:
         test: ["CMD", "curl", "-f", "http://localhost:7474"]
-        interval: 30s
+        interval: 10s
         timeout: 10s
-        retries: 5
+        retries: 10
   cartography:
-    image: lyft/cartography
+    image: ghcr.io/lyft/cartography:0.63.0
```

Comment:
No need to list the specific version. If it's unlisted, it'll use the latest released tag, which will let us avoid having to update this version tag everytime we cut a release.

Reply:
Done @ryan-lane please merge.